app-frontend/src/app/pages/projects/edit/scenes/scenes.module.js

```diff
@@ -94,6 +100,15 @@ class ProjectsScenesController {
             this.$state.go('projects.edit.browse', {sceneid: null, bbox: bbox.toBBoxString()});
         });
     }
+
+    checkPendingImports() {
+        this.uploadService.query({
+            uploadStatus: 'UPLOADED',
+            projectId: this.projectId
+        }).then(uploads => {
+            this.pendingImports = uploads.results.length;
```

Comment:
this is a paginated endpoint - use `uploads.count` and a page size of 1 to reduce query time